src/blocks/scratch3_sensing.js

```diff
@@ -99,4 +101,11 @@ Scratch3SensingBlocks.prototype.getKeyPressed = function (args, util) {
     return util.ioQuery('keyboard', 'getKeyIsDown', args.KEY_OPTION);
 };
 
+Scratch3SensingBlocks.prototype.daysSince2000 = function()
+{
+  var start = new Date(2000, 1-1, 1); 
```

Comment:
Probably to indent it like this: `****` (where each `*` is a space) instead of like this: `**`.